zbus/src/address.rs

```diff
@@ -168,6 +168,10 @@ pub enum Address {
     /// A listenable address using a specified path, in which a socket file with a random file name
     /// starting with 'dbus-' will be created by the server.
     UnixDir(OsString),
+    /// The same as UnixDir, except that on platforms with abstract sockets, the server may attempt
```

Comment:
* may? Isn't it that it **will** make the attempt?
* how do clients figure out the full path (same with `UnixDir`)? Probably should document that.

Reply:
It's from the spec. It doesn't have to, so it may.. 

Clients can't figure out the path themself, the server or the user is providing it.

Reply:
It may use non-abstract path. The server typically communicate the address with --print-address (see https://github.com/dbus2/busd/pull/23/commits/f85e20473077f1ed0e009860d6b3a5d6641f699c)

Reply:
> It may use non-abstract path. The server typically communicate the address with --print-address

But then the actual path is part of the address then. The docs are just confusing in that case.

Reply:
What is confusing exactly? Can you suggest a different documentation then?

Reply:
> What is confusing exactly?

It's not clear that the actual path is part of the address. Reading the docs, you get the impression that it's just random. Hence my initial question/confusion.

Reply:
added